samples/SocketsSample/wwwroot/sockets.html

```diff
@@ -0,0 +1,47 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+<body>
+    <h1 id="transportName">Unknown Transport</h1>
+
+    <form id="sendmessage">
+        <input type="text" id="data" />
+        <input type="submit" value="Send" />
+    </form>
+
+    <ul id="messages"></ul>
+    <script src="lib/signalr-client/signalr-client.js"></script>
+    <script src="utils.js"></script>
+    <script>
+        document.addEventListener('DOMContentLoaded', () => {
+            let transport = getParameterByName('transport') || 'webSockets';
+
+            document.getElementById('transportName').innerHTML = transport;
+
+            var url = `http://${document.location.host}/chat`
```

Comment:
Yeah, since we're already using other ES6 features, and this is an internal sample, I suppose that's fine :)